pkg/cidata/user-data.TEMPLATE

```diff
@@ -67,6 +69,7 @@ write_files:
    # We do not use per-once.
    path: /var/lib/cloud/scripts/per-boot/00-base.boot.sh
    permissions: '0755'
+{{- if .Mounts}}
```

Comment:
Even when `len(Mounts)==0`, `uidmap` (`shadow-utils`) has to be installed when `Containerd.User == true`

Reply:
I believe I've fixed this now